### ISSUE TITLE
rails aborted! Unknown version "7.0"

### ISSUE THREAD

**Issue by cesc1989** [devaspros/puntapie#4]
```
Bundle complete! 30 Gemfile dependencies, 105 gems now installed.
Use `bundle info [gemname]` to see where a bundled gem is installed.
         run  bundle binstubs bundler
       **rails  importmap:install** <- notice this
rails aborted!
Unknown version "7.0"
/Users/fquintero/.rvm/gems/ruby-2.7.1/gems/railties-6.1.3.2/lib/rails/application/configuration.rb:205:in `load_defaults'
/Users/fquintero/projects/devaspros-projects/prueba_puntapie/config/application.rb:24:in `<class:Application>'
/Users/fquintero/projects/devaspros-projects/prueba_puntapie/config/application.rb:23:in `<module:PruebaPuntapie>'
/Users/fquintero/projects/devaspros-projects/prueba_puntapie/config/application.rb:21:in `<top (required)>'
/Users/fquintero/projects/devaspros-projects/prueba_puntapie/Rakefile:4:in `require_relative'
/Users/fquintero/projects/devaspros-projects/prueba_puntapie/Rakefile:4:in `<top (required)>'
/Users/fquintero/.rvm/gems/ruby-2.7.1/gems/railties-6.1.3.2/lib/rails/commands/rake/rake_command.rb:20:in `block in perform'
/Users/fquintero/.rvm/gems/ruby-2.7.1/gems/railties-6.1.3.2/lib/rails/commands/rake/rake_command.rb:18:in `perform'
/Users/fquintero/.rvm/gems/ruby-2.7.1/gems/railties-6.1.3.2/lib/rails/command.rb:52:in `invoke'
/Users/fquintero/.rvm/gems/ruby-2.7.1/gems/railties-6.1.3.2/lib/rails/commands.rb:18:in `<top (required)>'
bin/rails:4:in `require'
bin/rails:4:in `<main>'
```

The error could be because it's doing some importmap stuff but it's rails 6 not 7!

**Comment by cesc1989** [comment]
The thing is for some reason the generator sets `config.load_defaults` to version 7.0 in `config/application.rb`:

```
config.load_defaults 7.0
```

I inspected source code and it looks OK. Don't know how to send the correct version to the generator. Tried adding a `RAILS_VERSION` file and also placing a string in a `rails_version` method in the template.rb but it doesn't work.

What seems to work is using rails 7 in Gemfile.tt
